### PR TITLE
add border

### PR DESCRIPTION
**Pull request by joanlmGit**
border

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 body {
-    background-color: aquamarine;
+    background-color: rgb(203, 24, 209);
+    border: solid 3px rgb(95, 24, 209);
```
Comment (Aitor1979) [inline style.css]: man, is not fucsia, should be "pink" color

Comment (joanlmGit) [reply style.css]: Ok, chenge yet

File: style.css [hunk 2]
```diff
@@ -1,5 +1,6 @@
 body {
```
Comment (Aitor1979) [inline style.css:1]: Ok

**Comment by joanlmGit** [comment]
esta corregido

**Pull request merged**